vortex-duckdb/src/exporter/dict.rs

```diff
@@ -63,7 +72,11 @@ pub(crate) fn new_exporter(
 impl<I: NativePType + AsPrimitive<u32>> ColumnExporter for DictExporter<I> {
     fn export(&self, offset: usize, len: usize, vector: &mut Vector) -> VortexResult<()> {
         // Copy across the dictionary values.
-        vector.reference(&self.values_vector);
+        let ref_vector = self
+            .values_vector
+            .lock()
+            .vortex_expect("error: failed to get lock");
```

Comment:
Use parkinglot::Mutex and you won't have an error.

Reply:
done